packages/react-app/src/components/AddressInput.jsx

```diff
@@ -1,29 +1,25 @@
 import { CameraOutlined, QrcodeOutlined } from "@ant-design/icons";
-import { Badge, Input } from "antd";
-import { useLookupAddress } from "eth-hooks/dapps/ens";
```

Comment:
I was just starting the ethereumSpeedRun and found out that this change it's braking my fresh build D: